### ISSUE TITLE
Could not connect to the container. Do you have sufficient privileges?

### ISSUE THREAD

**Issue by marlon617** [openshift/openshift-ansible#11590]
version:
```
# oc version
oc v3.11.0+62803d0-1
kubernetes v1.11.0+d4cacc0
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://openshift-cluster.server.gingkoo:8443
openshift v3.11.0+9b1e777-164
kubernetes v1.11.0+d4cacc0
```

Could not connect to the container. Do you have sufficient privileges?

新增节点完成后。
![image](https://user-images.githubusercontent.com/5423846/57354989-98defc80-719f-11e9-8e95-837ed73cf982.png)

**Comment by marlon617** [comment]
将master集群重启后正常

**Comment by Caplost** [comment]
没用

**Comment by bk203** [comment]
Had the same issue and had a hard time trying to find where the 500 exception was logged. This error would only show on containers on a newly added node. After some debugging I found trying to connect to the container using `oc rsh` gave a more detailed error. The master could not resolve the hostname of the new node. This can be verified from every master api trying to `ping` the new host by hostname. I our case we use the system hosts file and not a separate DNS server so we had to update one of our masters hosts file with the new host and restarting the master api using the `master-restart api`. Also recommend checking the controllers and etcd while you're at it.
